packages/desktop/components/popups/AliasConfirmationPopup.svelte

```diff
@@ -42,9 +38,10 @@
     $: void setStorageDeposit(aliasOutput)
     $: isTransferring = $selectedAccount.isTransferring
 
-    async function setStorageDeposit(aliasOutput): Promise<void> {
+    async function setStorageDeposit(aliasOutput: AliasOutput): Promise<void> {
         try {
-            const { amount } = await $selectedAccount.buildAliasOutput(aliasOutput)
+            const client = await getClient()
+            const { amount } = await client.buildAliasOutput(aliasOutput)
```

Comment:
Is this working for you like this?

Reply:
Yeah, looks like its working correctly. Are you experiencing some issues here?
![image](https://github.com/iotaledger/firefly/assets/52735957/b20f2097-613a-4c35-875e-d7d29b57f97e)

Reply:
Yeah, I had to add an `amount` field to `aliasOutput`. It was erroring otherwise. Not sure why it doesn't for Firefly.